### DISCUSSION TITLE
ies_subset_how effects

### DISCUSSION THREAD

**Discussion by RyanConway91** [usgs/pestpp#211] [Q&A]
I came across the ies_subset_how parameter in the documentation. Would using phi_based produce bias in the parameter estimates compared to random?

**Comment by jtwhite79** [comment]
You know that whole subset thing is a short cut and there are always consequences for short cuts.  Lately I tend to use the random subset and pass the subset size as -10, which means run 10% of the realizations in the subset testing.  This seems to help make sure that the selected lambda and scale factor are representative of the entire ensemble, which I think is the ultimate goal of the subset testing...
